src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,22 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+import java.math.BigDecimal;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) INJECTOR.getInstance(BookDao.class);
+        Book book = new Book("GOT", BigDecimal.valueOf(140L));
+        System.out.println(bookDao.create(book));
+        bookDao.findAll();
+        System.out.println(bookDao.findById(1L));
```

Comment:
how to you know that id of just created book is 1 or that there are some id with value 1?